.eslintrc.json

```diff
@@ -30,6 +30,6 @@
     "no-loop-func": "off",
     "import/extensions": "off",
     "import/prefer-default-export": "off",
-    "eslint-comments/no-unused-disable": "error"
+    "eslint-comments/no-unused-disable": "off"
```

Comment:
Either this or we should specifically disable import/no-unresolved and remove all those lint lines.

Reply:
Added a commit to that effect. I think it will be better to be able to notice other unused directives early.

Reply:
yeah my hunch is that disabling `import/no-unresolved` is an artifact of a weird build/lint process, or of some confusion between eslint and node's import-lookup mechanism (ideally eslint would look at `package.json` to see what is declared as available, rather than `node_modules`). With the exception of modules that are provided by a SES/swingset-like virtualized environment, I'm thinking that linting should pass without complaint regardless of whether `yarn install` has been run or not (maybe that's unrealistic). Maybe at the very least it should not be necessary to *uninstall* everything before running lint, that's a painful hammer to apply.